services/github/github-issue-detail.service.js

```diff
@@ -30,7 +140,7 @@ module.exports = class GithubIssueDetail extends LegacyService {
     return {
       base: 'github/issues/detail',
```

Comment:
I left the route here as it was previously defined, but wanted to highlight that the Legacy Service supported both `issues` and `pulls` (`github/issues|pulls/detail`). Is the desire to narrow the official path to just be `issues`?

If so I'll add a redirector service to prevent breaking anyone using `github/pulls/detail/...`

Reply:
I'm fine with that.

Reply:
Done!